draft-ietf-lpwan-schc-over-lorawan.md

```diff
@@ -801,42 +799,52 @@ ruleID, 21 bits residue + 279 bytes payload.
 |   1    |       21 bits       | 279 bytes |
 
 
-The current LoRaWAN MTU is 11 bytes, although 2 bytes FOpts are used by
-LoRaWAN protocol: 9 bytes are available for SCHC payload + 1 byte FPort
-field.  SCHC header is 2 bytes (including FPort) so 1 tile is sent in
-first fragment.
+The current LoRaWAN MTU is 11 bytes, 0 bytes FOpts are used by LoRaWAN
+protocol: 11 bytes are available for SCHC payload + 1 byte FPort field.
+SCHC header is 2 bytes (including FPort) so 1 tile is sent in first
+fragment.
 
-| LoRaWAN Header                        | LoRaWAN payload (6 bytes) |
-+ ------------------------------------- + ------------------------- +
-|                |  FOpts  | RuleID=20  |   W   |  FCN   |  1 tile  |
-+ -------------- + ------- + ---------- + ----- + ------ + -------- +
-|       XXXX     | 2 bytes | 1 byte     | 0   0 |   62   | 5 bytes  |
+| LoRaWAN Header                        | LoRaWAN payload (11 bytes) |
++ ------------------------------------- + -------------------------- +
+|                |  FOpts  | RuleID=20  |   W   |  FCN   |  1 tile   |
++ -------------- + ------- + ---------- + ----- + ------ + --------- +
+|       XXXX     | 2 bytes | 1 byte     | 0   0 |   62   | 10 bytes  |
 
 
 Content of the tile is:
 | RuleID | Compression residue |  Payload          |
 + ------ + ------------------- + ----------------- +
-|   1    |       21 bits       |  1 byte + 3 bits  |
+|   1    |       21 bits       |  6 byte + 3 bits  |
+
+
+Next transmission MTU is 11 bytes, although 2 bytes FOpts are used by
+LoRaWAN protocol: 9 bytes are available for SCHC payload + 1 byte FPort
+field, a tile does not fit inside so LoRaWAN stack will send only FOpts.
 
+| LoRaWAN Header                        | 
++ ------------------------------------- +
+|                |  FOpts  |  FPort=20  |
++ -------------- + ------- + ---------- +
+|       XXXX     | 2 bytes | 1 byte     |
 
```

Comment:
Are you sure FPort is required when no payload is sent? How will the SCHC gateway react with an empty payload, and just a RuleID = 20 ? (SCHC gateway will expect W / FCN field)

I suggest to remove the example table and just keep the text (text is very clear)

Reply:
You are right, that is a mistake. I pretty sure I deleted the 2 here to have FPort=0...
Anyway you are right, text is explicit enough